src/main/java/org/dimdev/riftloader/RiftLoader.java

```diff
@@ -49,6 +50,17 @@ public void load(boolean isClient) {
 
         side = isClient ? Side.CLIENT : Side.SERVER;
 
+        // test if mods/"version"-Rift contains any .jar files
+        modsVersionSpecifiedDir.mkdir();
```

Comment:
Well, there are two reasons:
1. If folder `mods/_version_-Rift` does not exists, the checking will throw an exception. I just want to keep it simple
2. A special folder for Rift is better to separate between Rift's and forge's mods (or any other APIs). Since lots of ppl will just install multiple types of API together without a second thought. And guys like me would like to play different versions at the same time. So this new folder will make life much more easier without dealing too much with conflicts.

I'm still looking for solution to automate the `_version_` part. Either pull from `RealmsSharedConstants.VERSION_STRING` like you addressed in #43 . Or coded with `build.gradle` "replace" function. What's your suggestion?